src/Compatibility/Core/src/MauiHandlersCollectionExtensions.cs

```diff
@@ -6,11 +6,23 @@ namespace Microsoft.Maui.Controls.Compatibility
 {
 	public static class MauiHandlersCollectionExtensions
 	{
+		public static IMauiHandlersCollection TryAddCompatibilityRenderer(this IMauiHandlersCollection handlersCollection, Type controlType, Type rendererType)
+		{
+			// This will eventually get copied to the all the other adds once we get rid of FormsCompatBuilder
+			Internals.Registrar.Registered.Register(controlType, rendererType);
```

Comment:
I had this initially using FormsCompatBuilder and it wasn't working because FormsCompatBuilder only runs through and registers the `PendingRenderers` if you call "UseFormsCompat" but I changed all this code so we're not longer calling 'UseFormsCompat' anymore